tests/test_snmpserver.rb

```diff
@@ -312,9 +312,9 @@ def test_snmpserver_global_enforce_priv_get_enabled
     snmpserver.global_enforce_priv = true
     device_enabled = snmpserver.global_enforce_priv?
     if device_enabled
-      refute_show_match(pattern: /no snmp-server globalEnforcePriv/)
+      assert_show_match(pattern: /snmp-server globalEnforcePriv/)
     else
-      assert_show_match(pattern: /no snmp-server globalEnforcePriv/)
+      refute_show_match(pattern: /no snmp-server globalEnforcePriv/)
```

Comment:
Good points. It looks like this test was a cut&paste of the test before it (a default test) which also has the dubious logic. I think the intent they were going for in the first test was to set it to default (from the yaml instead of an explicit boolean) and then be able to test the pattern in either case. This test just copied the logic. I'll just rewrite these two tests.